docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -1000,6 +1000,16 @@ Sorbet:
   DocumentationExtension: .md
 ----
 
+== Setting the version tracking metadata for cops
+
+Each cop should gain the following additional metadata:
+
+* `VersionAdded` - the RuboCop version in which it was added
+* `VersionChanged` - the RuboCop version in which it was changed in a user-impacting way (new config, updated defaults, etc)
+* `VersionRemoved` - the RuboCop version in which the cop was removed for whatever reason.
```

Comment:
We don't seem to have used `VersionRemoved` anywhere, and it is not referenced anywhere other than being a valid configuration parameter. Perhaps it's useful to explain why it would be set? Why would we retain config for a cop that is removed?

Reply:
`VersionRemoved` was removed because it cannot be configured in `default.yml` and I added it along with a setting example for illustration.